pages/sign-in.js

```diff
@@ -1,85 +1,130 @@
-                        officia. Salutantibus vitae elit libero, a pharetra augue.
-                      </p>
+      <div className="max-w-screen-2xl">
+        <section className="lg:bg-amcovad-primary-500 overflow-hidden max-w-[84rem] ">
+          <div className=" bg-cover pb-24 lg:pb-0 bg-[url('../assets/images/md-hexagons.png')] bg-top bg-opacity-20 lg:bg-[url('../assets/images/hexagons.png')]">
+            <MnavBar />
+            <div className="flex ">
+              <div className="flex flex-col justify-center flex-1 px-4 py-8 sm:px-6 lg:flex-none lg:px-20 xl:px-24 ">
+                <div className="w-full max-w-[43rem] lg:w-[38rem]  mx-auto pt-5 lg:pt-0 ">
+                  <div className="relative lg:py-10 lg:px-8 lg:my-4 lg:max-w-[38rem] lg:w-[580px]">
+                    <div className=" hidden lg:block absolute left-3 top-[-8px]">
+                      <Image src={LogoBlack} alt="black logo" />
                     </div>
-                    <div className="mt-8">
-                      <form action="#" method="POST" className="space-y-1">
-                        <div className="">
+                    <div className="bg-amcovad-secondary-100 p-6 lg:p-8 ">
+                      <div>
+                        <h2 className=" font-RozhaOne max-w-[300px] md:max-w-[424px] text-2xl md:text-4xl font-extrabold lg:font-bold text-amcovad-tertiary pb-2">
+                          Get your transactions covered and secured.
+                        </h2>
+                        <p className="text-amcovad-secondary-700 text-[14px]">
+                          Gallia est omnis divisa in partes tres, quarum. Fabio vel iudice vincam, sunt in culpa qui
+                          officia. Salutantibus vitae elit libero, a pharetra augue.
+                        </p>
+                      </div>
+                      <div className="mt-8">
+                        <form onSubmit={handleSubmit(onSubmit)} className="space-y-1">
                           <div className="">
-                            <Input label="Email address" name="email" type="email" required placeholder=" " />
-                          </div>
-                          <div>
-                            <Input
-                              label="Password"
-                              name="password"
-                              type="password"
-                              required
-                              placeholder=" "
-                              Icon={<PasswordCheckIcon width="18px" height="18px" fill="#292D32" />}
-                            />
+                            <div className="">
+                              <Input
+                                label="Email address"
+                                name="email"
+                                type="email"
+                                register={register('email', { required: true })}
```

Comment:
I think it should be safe to remove the  `{ required: true }` since we are using yup for validation.

```suggestion
                                register={register('email')}
```